pkg/kubectl/delete.go

```diff
@@ -82,9 +80,6 @@ func ReaperFor(kind schema.GroupKind, c internalclientset.Interface, sc scalecli
 	case api.Kind("Pod"):
 		return &PodReaper{c.Core()}, nil
 
-	case batch.Kind("Job"):
```

Comment:
I'm not quite sure about this being removed already. I never can't find the guarantees we give wrt to backwards compatibility. @deads2k that reminds me we should probably clean the reapers, we have GC in for quite a while, but cli still manually handles these, which is bad imo.

Reply:
@soltysh I have found the website that describes deprecation policy https://kubernetes.io/docs/reference/deprecation-policy/

Reply:
removing code feels so good :)